handshake_ix.go

```diff
@@ -99,20 +99,7 @@ func ixHandshakeStage1(f *Interface, addr netip.AddrPort, via *ViaSender, packet
 		return
 	}
 
-	vpnIp, ok := netip.AddrFromSlice(remoteCert.Details.Ips[0].IP)
-	if !ok {
-		e := f.l.WithError(err).WithField("udpAddr", addr).
-			WithField("handshake", m{"stage": 1, "style": "ix_psk0"})
-
-		if f.l.Level > logrus.DebugLevel {
-			e = e.WithField("cert", remoteCert)
-		}
-
-		e.Info("Invalid vpn ip from host")
-		return
-	}
-
-	vpnIp = vpnIp.Unmap()
+	vpnIp := remoteCert.Details.Ip.Addr()
```

Comment:
If  `Ip` is somehow blank/invalid, do we need a check for that here to mirror the old way?